openpgp/s2k/s2k.go

```diff
@@ -80,6 +148,15 @@ func (c *Config) EncodedCount() uint8 {
 	return encodeCount(i)
 }
 
+func DefaultArgonConfig() *ArgonConfig {
+	aconf := &ArgonConfig{
+		NumberOfPasses:      3,
+		DegreeOfParallelism: 4,
+		MemoryExponent:      16, // 64 MiB of RAM
+	}
+	return aconf
+}
```

Comment:
Rather than creating a default argon2 config in one go, I would set defaults for the individual parameters, like is done for the other configs (with individual getter functions). That way, people can pass `&ArgonConfig{ MemoryExponent: 32 }` and still get the default value for the other two parameters, for example.

You could move `s2k.Config` and `Argon2Config` to a separate file (e.g. `s2k_config.go`) or even two separate files if you want, to not clutter this one too much with all the getter functions.